packages/labs/analyzer/test-files/ts/properties/src/element-a.ts

```diff
@@ -25,13 +25,29 @@ export class ElementA extends LitElement {
 
   declare staticProp: number;
 
+  declare constructorAssignOnly: number;
+
   constructor() {
     super();
     this.staticProp = 42;
+    this.constructorAssignOnly = 0;
```

Comment:
without being declared `readonly` is this any difference from `staticProp`?

Reply:
it's needed for the opposite case in the javascript file, since these are supposed to be mirror images